src/main/java/org/sagebionetworks/web/client/widget/entity/browse/EntityFilter.java

```diff
@@ -5,17 +5,29 @@
 import static org.sagebionetworks.repo.model.EntityType.folder;
 import static org.sagebionetworks.repo.model.EntityType.link;
 import static org.sagebionetworks.repo.model.EntityType.project;
+import static org.sagebionetworks.repo.model.EntityType.submissionview;
 import static org.sagebionetworks.repo.model.EntityType.table;
+
 import java.util.ArrayList;
 import java.util.HashSet;
 import java.util.List;
 import java.util.Set;
+
 import org.sagebionetworks.repo.model.EntityHeader;
 import org.sagebionetworks.repo.model.EntityType;
 import org.sagebionetworks.web.client.EntityTypeUtils;
 
 public enum EntityFilter {
-	ALL(project, folder, file, link), CONTAINER(project, folder), PROJECT(project), FOLDER(folder), FILE(file), ALL_BUT_LINK(project, folder, file), PROJECT_OR_TABLE(project, table, entityview);
+	ALL(EntityType.values()),
+	ALL_DIRECTORY(project, folder, file, link),
+	CONTAINER(project, folder),
+	PROJECT(project),
+	FOLDER(folder),
+	FILE(file),
+	TABLE(table),
+	ALL_DIRECTORY_BUT_LINK(project, folder, file),
+	ALL_TABLES(table, entityview, submissionview),
+	PROJECT_OR_TABLE(project, table, entityview, submissionview);
```

Comment:
Changing the meaning of "ALL" here to really mean all entity types. Open to naming suggestion for "ALL_DIRECTORY" which is now what "ALL" was

Reply:
Confused by this.  In my mind DIRECTORY is the same as CONTAINER, what do you mean by DIRECTORY?

Reply:
Before my changes, `ALL` was what we show in the EntityTree (projects, folders, files, links). `ALL` is obviously no longer a good name for this. It's kind of an arbitrary collection, so I'm not sure if there's a better name for it. For now, renaming to `PROJECT_FOLDER_FILE_LINK`